### PR TITLE
filepreview: Fix font color and style so text files are more readable

### PR DESCRIPTION
**Pull request by jzila**
cc @cecileboucheron

### CODE REVIEW

File: shared/fs/filepreview/text-view.desktop.js [hunk 1]
```diff
@@ -21,18 +21,19 @@ const TextView = (props: TextViewProps) => (
...
 body{
-  background-color: ${globalColors.white};
-  padding: ${globalMargins.xlarge};
+  background-color: ${Styles.globalColors.white};
+  padding: ${Styles.globalMargins.xlarge};
   margin: 0;
-  color: ${globalColors.black_50};
+  color: ${Styles.globalColors.black};
   line-height: 1.38;
   font-size: 13;
```
Comment (cecileboucheron) [inline shared/fs/filepreview/text-view.desktop.js]: I think we could do 14 here, I tried your branch and it's still very hard to read in 13.

**Comment by cecileboucheron** [comment]
👍

File: shared/fs/filepreview/text-view.native.js [hunk 2]
```diff
@@ -15,18 +15,20 @@ const TextView = ({url, onInvalidToken, onLoadingStateChange}: TextViewProps) =>
...
-  background-color: ${globalColors.white};
-  padding: ${globalMargins.medium};
+  background-color: ${Styles.globalColors.white};
+  padding: ${Styles.globalMargins.medium};
   margin: 0;
-  color: ${globalColors.black_50};
+  color: ${Styles.globalColors.black};
   font-size: 15;
   line-height: 1.6;
+  font-family: ${Styles.globalStyles.fontTerminal.fontFamily}
```
Comment (songgao) [inline shared/fs/filepreview/text-view.native.js]: Do we need to worry about whether this font is available from webview on systems where this font is not installed?

**Comment by jzila** [comment]
I'm going to merge this and figure out the font family thing later.

**Pull request merged**